tests/rhtap-demo/rhtap-demo.go

```diff
@@ -62,7 +62,7 @@ const (
 	releasePollingInterval    = time.Second * 1
 
 	// test metadata
-	devEnvTestLabel = "rhtap-demo"
+	devEnvTestLabel = "konflux"
```

Comment:
I think this change needs to be reflected in other files like `magefile.go`, rhtap-demo readme file etc.